### ISSUE TITLE
Build arm-image on x86 architecture

### ISSUE THREAD

**Issue by urpylka** [CopterExpress/clever#21]
Now we use chroot on Raspberry Pi 3 for build image.

> In this scope there are many problems:
> - absence of arm-architecture support on many clouds
> - low reliability
> - low machine capabilities (e.g. Jenkins web-UI slow work)
> - low resource of microsd-card

For use chroot on x86 for execute arm-binaries I propose use qemu-static interpreter as made in https://resin.io/blog/building-arm-containers-on-any-x86-machine-even-dockerhub/

**Comment by urpylka** [comment]
I tested build with qemu-static interpretator & encountered the problem "python OverFlow Error" in several places:
```
Traceback (most recent call last):
File "/usr/bin/apt-listchanges", line 37, in <module>
    import apt_listchanges, DebianFiles, ALCApt, ALCConfig, ALCSeenDb
File "/usr/share/apt-listchanges/apt_listchanges.py", line 32, in <module>
    import email.message
File "/usr/lib/python3.5/email/message.py", line 16, in <module>
    from email import utils
File "/usr/lib/python3.5/email/utils.py", line 30, in <module>
    import datetime
File "/usr/lib/python3.5/datetime.py“, line 638, in <module>
    microseconds=999999)
File "/usr/lib/python3.5/datetime.py", line 430, in __new__
    raise OverflowError("timedelta # of days is too large: %d" % d)

OverflowError: timedelta # of days is too large: 1000000000
```
```
Traceback (most recent call last):
File "/usr/bin/py3compile", line 34, in <module>
    from debpython.version import SUPPORTED, debsorted, vrepr,
File usr/share/python3/debpython/__init__.py", line 2, in <module>
    from datetime import datetime
File "/usr/lib/python3.5/datetime.py", line 638, in <module>
    microseconds=999999)
File "/usr/lib/python3.5/datetime.py", line 430, in __new__
    raise OverflowError("timedelta # of days is too large: %d" % d)

OverflowError: timedelta # of days is too large: 1000000000
```
I reproduced this bug in chroot & tested on host OS
```
[urpylka@ubuntu:~/chroot$ sudo chroot . /bin/sh
[# python
Python 2.7.13 (default, Nov 24 2017, 17:33:09)
[GCC 6.3.0 20170516] on linux2
Type "help", "copyright", "credits" or "license" for more information.
import datetime
[>>>
[# python3
Python 3.5.3 (default, Jan 19 2017, 14:11:04)
[GCC 6.3.0 20170124] on linux
Type "help", "copyright", "credits" or "license" for more information.
[>>> import datetime
Traceback (most recent call last):
File "<stdin>", line 1, in <module>
File "lusr/lib/python3.5/datetime.py", line 638, in <module>
    microseconds=999999)
File "lusr/lib/python3.5/datetime.py“, line 430, in __new__
    raise overflowError("timedelta # of days is too large: %d" % d)

OverflowError: timedelta # of days is too large: 1000000000
[#
[urpylka@ubuntu:~/chroot$ python3
Python 3.5.2 (default, Nov 17 2016, 17:05:23)
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
[>>> import datetime
```
and files /usr/lib/python3.5/datetime.py on chroot & host OS is same.

**Comment by urpylka** [comment]
For reproduce this error on your PC you will need:

Image of Raspbian such as `2017-11-29-raspbian-stretch-lite.img` you can get by:
```bash
wget -nv -O raspbian_lite_latest.zip https://downloads.raspberrypi.org/raspbian_lite_latest
unzip -uo ./raspbian_lite_latest.zip -d .
```
Static interpreter `qemu-arm` you can get by:
```bash
git clone git://git.qemu.org/qemu.git  
cd qemu  
./configure --target-list=arm-linux-user --static
make  
```

Then enter `sudo su` and copy-past code below:
```bash
# Mount image
losetup -Pf 2017-11-29-raspbian-stretch-lite.img
mount /dev/loop0p2 /mnt
mount /dev/loop0p1 /mnt/boot

# Enable arm arhitecture
cp qemu/arm-linux-user/qemu-arm /mnt/usr/bin/qemu-arm-static
mount binfmt_misc -t binfmt_misc /proc/sys/fs/binfmt_misc  
echo ':arm:M::\x7fELF\x01\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x28\x00:\xff\xff\xff\xff\xff\xff\xff\x00\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff:/usr/bin/qemu-arm-static:' > /proc/sys/fs/binfmt_misc/register

# Test
chroot /mnt /bin/sh -c "python3.5 -c 'import datetime'"
```

**Comment by urpylka** [comment]
Temporarily, I solved this problem by replacing  `999999999` to `1000000000` in https://github.com/CopterExpress/clever/commit/4e922600a2517b1805b1270a8e80466b68fad9cc

**Comment by urpylka** [comment]
Ok, we made worked prototype & we can use this for build image on x86 or dockerhub after improvement.
